CMakeLists.txt

```diff
@@ -202,6 +202,11 @@ target_include_directories(
   dosbox PUBLIC include ${CMAKE_CURRENT_BINARY_DIR}/include
 )
 
+# Disable cl/clangcl warnings about insecure C functions
+if(MSVC)
+  add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
+endif()
```

Comment:
Can we move this block to line 28, please?

Reply:
Sure, I think so.

It was originally there, because I first tried `target_compile_definitions`, and needed to be after the definition of the dosbox target.